.travis.yml

```diff
@@ -1,28 +1,40 @@
 language: ruby
 rvm:
-- 2.2.5
-- 2.5.1
+- 2.6
+- 2.5
+- 2.4
```

Comment:
Does this mean that we're dropping support for ruby < 2.4?

Reply:
Yeah, mostly I just didn't know how far back we should test. I'm happy to add more.